app/src/main/java/com/gdsctsec/smartt/ui/main/TTSchedulingScreenFragment.kt

```diff
@@ -24,7 +30,23 @@ class TTSchedulingScreenFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_t_t_scheduling_screen, container, false)
+        val view:View= inflater.inflate(R.layout.fragment_t_t_scheduling_screen, container, false)
+
+        val title: TextView =view.findViewById(R.id.remindi_text_view);
+        val word: Spannable = SpannableString("Rem")
+        word.setSpan(ForegroundColorSpan(Color.BLACK),0,word.length,Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+
+        title.setText(word)
+
+        val wordTwo:Spannable=SpannableString("i")
+        wordTwo.setSpan(ForegroundColorSpan(ContextCompat.getColor(requireContext(),R.color.remindi_icon_i)),0,wordTwo.length,Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+        title.append(wordTwo)
+
+        val wordThree:Spannable=SpannableString("ndi")
+        wordThree.setSpan(ForegroundColorSpan(Color.BLACK),0,wordThree.length,Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+        title.append(wordThree)
+
+        return view;
     }
```

Comment:
move all the UI code to `onViewCreated`, read [this](https://stackoverflow.com/questions/25119090/difference-between-oncreateview-and-onviewcreated-in-fragment) for more information